Dockerfile

```diff
@@ -167,6 +169,22 @@ RUN apk update && apk add \
     && chown -R www-data:www-data /home/www-data/ /var/www/html \
     && rm /var/www/html/install_composer.php
 
+
+# ------------------------ xdebug ------------------------
+ARG APP_ENV=production
+ARG REMOTE_XDEBUG_HOST=0.0.0.0
+ARG REMOTE_XDEBUG_PORT=9999
+RUN pecl install xdebug-2.9.1; \
+    docker-php-ext-enable xdebug; \
```

Comment:
I _love_ the idea of adding this to the container, thanks for submitting this PR!  I'd really like to keep the container as production-ready as possible.  What do you think about adjusting this so that it only installs xdebug if a few environment variables are set within docker-compose.  Something like...

```
services:
  api:
    image: 'my-image'
    environment:
     - XDEBUG_ENABLE=true
```

Also enabling `REMOTE_XDEBUG_HOST` and `REMOTE_XDEBUG_PORT` to be configured that way as well.

Reply:
We could also only enable xdebug at runtime, via an environment variable in the fpm bootup, [similar to how we do opcache](https://github.com/bkuhl/fpm-nginx/blob/master/services.d/php-fpm/run#L4).  That would allow someone to install it, but disable it if it causes performance issues with their app with a simple container reboot rather than rebuilding the container.

Reply:
Yeah, that should be our goal to make the container always production ready. 
In fact in my first [commit](https://github.com/RafikFarhad/fpm-nginx/blob/b57b05da656bd229e7174ceae86d067452d2de95/Dockerfile#L175) I checked the $APP_ENV before installing `xdebug`. Are you suggesting that? 
Secondly: Are you suggesting that `xdebug` should be enabled based on `XDEBUG_INSTALL` env? I'm confused because `xdebug` is currently [activated depending on](https://github.com/RafikFarhad/fpm-nginx/blob/488721f5699267c676bdc4631a61124e98c3ad76/services.d/php-fpm/run#L4) the APP_ENV value, only in `dev`. I think we can use if `XDEBUG_INSTALL` env.

Reply:
> In fact in my first commit I checked the $APP_ENV before installing xdebug. Are you suggesting that?

Yeah I think that would work.  Maybe we should support a few variations for flexibility: `dev`, `develop`, `development`, `local`?  Actually, it'd probably be better to check XDEBUG_ENABLE instead to give folks full control.

> Secondly: Are you suggesting that xdebug should be enabled based on XDEBUG_INSTALL env? I'm confused because xdebug is currently activated depending on the APP_ENV value, only in dev. I think we can use if XDEBUG_INSTALL env.

Sorry I didn't even see your changes to `php-fpm/run` with my initial review - trying to get my kids ready for the day at the same time.  I think it's valuable to give people the option to install it and enable it separately so locally they can leave it installed but selectively choose when to enable by restarting their container.  If `XDEBUG_ENABLE` controls whether it's enabled, I think we're ok installing it for all non-production environments.  What do you think?
I work on a large app at work that runs _super_ slow when xdebug is enabled and it wouldn't be great if I had to rebuild the container to control whether xdebug is enabled.